app/src/main/java/pm/gnosis/heimdall/data/repositories/BridgeRepository.kt

```diff
@@ -5,18 +5,21 @@ import io.reactivex.Observable
 import io.reactivex.Single
 import pm.gnosis.model.Solidity
 
+
+typealias SessionIdAndSafe = Pair<String, Solidity.Address>
+
 interface BridgeRepository {
     data class SessionMeta(
         val id: String,
         val dappName: String?,
         val dappDescription: String?,
+        val dappUrl: String?,
         val dappIcons: List<String>?,
         val active: Boolean,
         val approvedSafes: List<Solidity.Address>?
```

Comment:
maybe better to remove nullability for approvedSafes and add helper function/property hasApprovedSafes. Like val hasApprovedSafes = approvedSafes.isNotEmpty()

Reply:
on low level there is a slight difference between `null` and emptyList here.

Currently we are not using the parameter, so I would rather not change it for now.